### PR TITLE
solution

### PR DESCRIPTION
**Pull request by DianaKyrychuk**

### CODE REVIEW

File: src/isPasswordActual.test.js [hunk 1]
```diff
@@ -8,13 +8,27 @@ describe(`Function 'isPasswordActual':`, () => {
   });
 
   it(`should return a string`, () => {
```
Comment (coyotious) [inline src/isPasswordActual.test.js:10]: I would change this for:
expect(typeof isPasswordActual()).toBe('string');

File: src/isPasswordActual.test.js [hunk 2]
```diff
@@ -8,13 +8,27 @@ describe(`Function 'isPasswordActual':`, () => {
...
+    expect(isPasswordActual(2013, 6, 9))
+      .toBe('Immediately change the password!');
+  });
 
+  it(`should ask to change password if was changed > 1 year ago`, () => {
+    expect(isPasswordActual(2020, 7, 19))
+      .toBe('Immediately change the password!');
   });
```
Comment (coyotious) [inline src/isPasswordActual.test.js:18]: Need more tests for boundary values: 60, 59, 30, 29 (days ago).

File: src/isPasswordActual.test.js [hunk 3]
```diff
@@ -31,4 +30,24 @@ describe(`Function 'isPasswordActual':`, () => {
...
+
+  it(`shouldn't ask to change password if was changed 29 days ago`, () => {
+    expect(isPasswordActual(2021, 6, 21))
+      .toBe('Password is actual.');
+  });
+
+  it(`should ask to change password if was changed 59 days ago`, () => {
+    expect(isPasswordActual(2021, 5, 22))
+      .toBe('You should change your password.');
+  });
```
Comment (tnboyko) [inline src/isPasswordActual.test.js:52]: давай добавим проверку если пароль был изменен сегодня

**Review by tnboyko — CHANGES_REQUESTED** [review]
